examples/plot_vector_testing.py

```diff
@@ -0,0 +1,205 @@
+multi_line_gdf["attr"] = ["road", "stream"]
+
+# Create a point GeoDataFrame
+
+points = pd.DataFrame(
+    {
+        "lat": np.array([1, 2, 1, 0, 4]),
+        "lon": np.array([3, 4, 0, 0, 1]),
+    }
+)
+
+point_gdf = gpd.GeoDataFrame(
+    {"A": np.arange(5), "B": np.arange(5)},
+    geometry=gpd.points_from_xy(points.lon, points.lat), crs="epsg:4326"
+)
+point_gdf["size"] = [100, 100, 300, 300, 500]
+
+# Create symbology dictionary to use in the legend
+
+line_symb = {"road": "black", "stream": "blue"}
+point_symb = {100: "purple", 300: "green", 500: "brown"}
+
+################################################################################
+# Create Your Spatial Plot
+# ------------------------
+# Above you created several GeoPandas GeoDataFrame objects that you want
+# to plot. To plot these data according to attribute value, you can group
+# the geometry by attributes and plot within a loop. Once you've created
+# your plot, you are ready to test it using MatPlotCheck.
+
+# Plot your data
+fig, ax = plt.subplots()
+polygon_gdf.plot(ax=ax, color="purple")
+
+# Plot your line data by attributes using groupby
+for ctype, lines in multi_line_gdf.groupby('attr'):
+    color = line_symb[ctype]
+    label = ctype
+    lines.plot(color=color, ax=ax, label=label)
+
+# Plot your points data by size using groupby
+for ctype, points in point_gdf.groupby('size'):
+    color = point_symb[ctype]
+    label = ctype
+    points.plot(color=color, ax=ax, label=label, markersize=ctype)
+
+# Add a legend
+ax.legend(title="Legend", loc=(1.1, .1));
+
+################################################################################
+# Create A MatPlotCheck VectorTester Object
+# -----------------------------------------
+# Once you've created your plot, you can create a MatPlotCheck VectorTester
+# object that can be used to test elements in the plot.
+
+vector_test = VectorTester(ax)
+
+###############################################################################
+#
+# .. note::
+#   Each geometry type must be tested seperately in VectorTester. So, if your
+#   plot has multiple geometry types, such as lines, polygons, and points,
+#   make sure to check each geometry type separately.
+
+###############################################################################
+#
+# .. note::
+#   Most tests are created as assert statements. Thus, if a test fails,
+#   matplotcheck will return an error. If the test passes, no message is
+#   returned.
+
+################################################################################
+# Test Point Attribute Values and Geometry xy Locations
+# -----------------------------------------------------
+# You can check that both the position of the points on the plot and the
+# associated point attribute values are
+# accurate using assert_points(), assert_points_grouped_by_type() and
+# assert_collection_sorted_by_markersize().
+#
+# To check the geometry locations, you can call assert_points() and prove the
```

Comment:
Is "prove" supposed to be "provide"?

Reply:
I just reworded this section to make more since. I don't think it was supposed to be provide either.